src/BlockTypes/ProductCategories.php

```diff
@@ -168,6 +168,10 @@ protected function fill_category_children( $categories, $categories_by_parent )
 	 * @return string Rendered output.
 	 */
 	protected function renderDropdown( $categories, $attributes, $uid ) {
+		$aria_label = empty( $attributes['hasCount'] ) ?
+			__( 'List of categories', 'woo-gutenberg-products-block' ) :
+			__( 'List of categories with their product counts', 'woo-gutenberg-products-block' );
```

Comment:
I like this solution so it's clear what the number means in the dropdown. :clap: